packages/transaction-controller/src/types.ts

```diff
@@ -524,6 +527,12 @@ export type TransactionBatchMeta = {
    */
   origin?: string;
 
+  /** Current status of the transaction. */
+  status: TransactionStatus;
+
+  /** When the transaction was created. */
+  time: number;
```

Comment:
Rather than adding these purely to support the `DefaultGasFeeFlow`, could we instead just use mock values at the point of calling? To minimise the complexity here as much as possible?